circle.yml

```diff
@@ -0,0 +1,23 @@
+machine:
+  php:
+    # https://circleci.com/docs/environment#php
+    version: 5.5.11
+  environment:
+    # DB config. Using default CircleCI's database.
+    TERMINUS_ENV: ci-$CIRCLE_BUILD_NUM
+    TERMINUS_SITE: wp-saml-auth
+
+dependencies:
+  cache_directories:
+    - ~/.composer/cache
+  override:
+    - composer global require pantheon-systems/terminus
+    - terminus auth login --machine-token=$TERMINUS_TOKEN
+
+test:
+  pre:
+    - echo 'pre'
```

Comment:
Any particular reason for including this file / these `echo` commands? I know we'll have CircleCI coming in https://github.com/pantheon-systems/wp-saml-auth/pull/21 too